server/controllers/rooms.js

```diff
@@ -29,6 +28,7 @@ module.exports = {
         });
 
     } else if (req.url === '/amenities') {
+      console.log('req.url:', req.url);
```

Comment:
Do you still need this console log?

Reply:
i deleted it i think i pushed not latest one ( i went back and forth branches before i will update and push again